groclient/client.py

```diff
@@ -850,7 +850,10 @@ def get_top(self, entity_type, num_results=5, **selection):
             self.access_token, self.api_host, entity_type, num_results, **selection
         )
 
-    def get_df(self, show_revisions=False, index_by_series=False):
+    def get_df(self,
+               show_revisions=False,
+               include_names=False,
```

Comment:
Also, could you update the docstring with a brief description of what include_names does (around line 871)?

Reply:
done